### PR TITLE
Add troubleshooting section about proxy squid cache

### PR DESCRIPTION
**Pull request by cbosdo**
# Description

Add a troubleshooting section describing how to clear the proxy squid cache in multiple configurations

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.3
- [ ] Manager-4.2

Cedric said: "backporting to 4.2 could be possible, but would require to remove all the container-related parts".

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/administration/pages/troubleshooting/tshoot-packages-proxy.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+kubectl delete pv/$PV_NAME
+----
+
+Deploy the pod again from the helm chart with the following command:
+
+----
+helm upgrade --install oci://...
+----
+
+Mind that this command uses `upgrade --install`.
```
Comment (keichwa) [inline modules/administration/pages/troubleshooting/tshoot-packages-proxy.adoc:60]: We usually use the simple expression "note ...".

```suggestion
Note that this command uses [command]``upgrade --install``.
```

**Review by keichwa — APPROVED** [review]

**Comment by keichwa** [comment]
@0rnela can you please check whether it confirms with you filename conventions and the title looks nice in the nav table?

Then we must add it to the nav file.

File: modules/administration/pages/troubleshooting/tshoot-packages-proxy.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/troubleshooting/tshoot-packages-proxy.adoc:60]: @cbosdo can you please also add a `.changelog` entry?

Comment (cbosdo) [reply modules/administration/pages/troubleshooting/tshoot-packages-proxy.adoc]: Done

Comment (keichwa) [reply modules/administration/pages/troubleshooting/tshoot-packages-proxy.adoc]: thank you!

**Review by keichwa — APPROVED** [review]

**Pull request merged**